### DISCUSSION TITLE
What is the GUI written in?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [twinbasic/twinbasic#1958] [Q&A] [answered]
Obviously, the editor is Monaco. However, is it running inside something like a Qt QWebView, or is it Electron, like VS Code is?

**Comment by WaynePhillipsEA** [comment, marked as answer]
It's a twinBASIC application, using the WebView2 control.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/twinbasic/twinbasic/discussions/1958#discussioncomment-11448895

@WaynePhillipsEA, Chromium Microsoft Edge's WebView2?

**Reply by WaynePhillipsEA** [reply]
Yes, that's right.
